### PR TITLE
Smoke test sample-configuration after deploy

### PR DESCRIPTION
**Pull request by giorgiosironi**

### CODE REVIEW

File: scripts/remote-deploy.sh [hunk 1]
```diff
@@ -28,3 +28,4 @@ if [ -n "$REVISION_DUMMY_API" ]; then
 fi
 
 docker-compose up --force-recreate -d
+.travis/smoke
```
Comment (giorgiosironi) [inline scripts/remote-deploy.sh]: should change to standard name once available

**Review by thewilkybarkid — APPROVED** [review]
Probably shouldn't be using something meant for Travis, but good enough for now.

**Comment by giorgiosironi** [comment]
Error with weird output:
```
+ docker-compose up --force-recreate -d
Recreating sample-configuration_dummy-api_fpm_1 ... 
Recreating sample-configuration_web_1           ... 
Recreating sample-configuration_browser_fpm_1   ... 
oke-test.sh
Smoke testing dummy-api
```

**Comment by giorgiosironi** [comment]
https://github.com/libero/sample-configuration/pull/5 to debug

**Pull request merged**